Gauge.VisualStudio.TestAdapter/GaugeRunner.cs

```diff
@@ -137,7 +137,17 @@ private void OnOutputDataReceived(object sender, DataReceivedEventArgs args)
                 return;
             using (var ms = new MemoryStream(Encoding.Unicode.GetBytes(args.Data)))
             {
-                var e = (TestExecutionEvent) serializer.ReadObject(ms);
+                TestExecutionEvent e;
+                try
+                {
+                    e = (TestExecutionEvent)serializer.ReadObject(ms);
+                }
+                catch (Exception err)
+                {
+                    _frameworkHandle.SendMessage(TestMessageLevel.Informational, $"Failed to deserialize : {args.Data}\n Error : {err}");
+                    MarkAllTestsFailed();
```

Comment:
Is it expected to mark all tests failed? or the current test skipped?

Reply:
We don't have information about the current test being executed as deserialization has failed.